src/base/bittorrent/sessionimpl.cpp

```diff
@@ -1382,7 +1382,7 @@ void SessionImpl::processNextResumeData(ResumeSessionContext *context)
         }
     }
 
-    Algorithm::removeIf(resumeData.tags, [this, &torrentID](const QString &tag)
+    erase_if(resumeData.tags, [this, &torrentID](const QString &tag)
```

Comment:
```suggestion
    std::erase_if(resumeData.tags, [this, &torrentID](const QString &tag)
```

Reply:
std::erase_if is only  for std containers.

Reply:
> std::erase_if is only for std containers.

I misread that it isn't std container.
Then as I said before I would prefer to use member `removeIf()`.

Reply:
>Then as I said before I would prefer to use member removeIf().

There are no such things.

Reply:
> > Then as I said before I would prefer to use member removeIf().
> 
> Well... there are no such things for set types.

Damn this Qt inconsistency!

Reply:
> Well... there are no such things for set types.

https://bugreports.qt.io/browse/QTBUG-115473

Reply:
> > Well... there are no such things for set types.
> 
> https://bugreports.qt.io/browse/QTBUG-115473

So according to people from Qt `QSet::removeIf()` exists but it's documentation that was missing.